.github/renovate.json5

```diff
@@ -72,7 +72,6 @@
         'cargo',
       ],
       matchPackageNames: [
-        'gix-features-for-configuration-only',
         'gix',
       ],
       groupName: 'gix',
```

Comment:
We can likely get rid of this group then.  This was just to make sure all gix packages were updated in the same PR